src/routes/github/github-router.ts

```diff
@@ -25,36 +39,35 @@ subRouter.post("/webhooks",
 	returnOnValidationError,
 	WebhookReceiverPost);
 
-//Have an cover all middleware to extract the optional gitHubAppId
-//subRouter.use(param("uuid").isUUID('all'), GithubServerAppMiddleware);
+// Create-branch is seperated above since it currently relies on query param to extract the jirahost
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
 
-// TODO: use jiraSymmetricJwtMiddleware and GithubServerAppMiddleware for all paths below once "create branch" supports JWT
-// subRouter.use(jiraSymmetricJwtMiddleware);
-// subRouter.use(GithubServerAppMiddleware);
+// OAuth Routes
+subRouter.use(GithubOAuthRouter);
+
+subRouter.use(jiraSymmetricJwtMiddleware);
+subRouter.use(GithubServerAppMiddleware);
 
 // OAuth Routes
-subRouter.use(jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubOAuthRouter);
+subRouter.use(GithubOAuthRouter);
 
 // CSRF Protection Middleware for all following routes
 subRouter.use(csrfMiddleware);
 
-subRouter.use("/setup", jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubSetupRouter);
+subRouter.use("/setup", GithubSetupRouter);
 
 // TODO: use GithubAuthMiddleware for all paths below once "create branch" supports JWT
 // subRouter.use(GithubAuthMiddleware);
 
 // App Manifest flow routes
-subRouter.use("/manifest", jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubManifestRouter);
+subRouter.use("/manifest", GithubManifestRouter);
 
-subRouter.use("/configuration", jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, GithubConfigurationRouter);
+subRouter.use("/configuration", GithubAuthMiddleware, GithubConfigurationRouter);
```

Comment:
**/nit**

So previously (before jwt middleware) the code is like 
```javascript
// All following routes need Github Auth
subRouter.use(GithubAuthMiddleware);

subRouter.use("/configuration", GithubConfigurationRouter);

// TODO: remove optional "s" once we change the frontend to use the proper delete method
subRouter.use("/subscriptions?", GithubSubscriptionRouter);

subRouter.use("/create-branch", GithubCreateBranchRouter);

subRouter.use("/repository", GithubRepositoryRouter);

subRouter.use("/branch", GithubBranchRouter);

```

Shall we mimic the same behaviour (which is the the `subRouter.use(GithubAuthMiddleware);` before the last couple routes?) <-- tbh I am not too sure about it. Personally I found it "safer" the extract `subRouter.use(GithubAuthMiddleware);`  into its own line. Burry into each routing is "less safe" to me.

Reply:
May as well finish the tidy while we are here lololol